### PR TITLE
add mailmap

### PR DESCRIPTION
**Pull request by jburel**
Add mailmap

3 entries that I cannot find info for 
* Steve Ogg (I can't find his current address)
* Felix cc (I remember felix but cannot remember his name)
* AarghIMApiratez ??
cc @joshmoore @sbesson 

This should allow to fix the entry in cla-audit

### CODE REVIEW

File: .mailmap [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+Petr Walczysko <p.walczysko@dundee.ac.uk> pwalczysko <p.walczysko@dundee.ac.uk>
+Riad Gozim <r.gozim@dundee.ac.uk> Riad Gozim <rgozim1@gmail.com>
+Roger Leigh <r.leigh@dundee.ac.uk> <rleigh@codelibre.net>
+Roger Leigh <r.leigh@dundee.ac.uk> <rleigh@debian.org>
+Roger Leigh <r.leigh@dundee.ac.uk> <rleigh@dundee.ac.uk>
+Sébastien Besson <sbesson@glencoesoftware.com> Sebastien Besson <seb.besson@gmail.com>
+Sébastien Besson <sbesson@glencoesoftware.com> Sébastien Besson <seb.besson@gmail.com>
+Sébastien Simard <ssimard@pasteur.fr> ssimard <ssimard@pasteur.fr>
+Simon Li <spli@dundee.ac.uk> <orpheus+devel@gmail.com>
+Simone Leo <s.z.leo@dundee.ac.uk> <simleo@crs4.it>
```
Comment (sbesson) [inline .mailmap:40]:
```suggestion
Simone Leo <s.z.leo@dundee.ac.uk> <simleo@crs4.it>
Tom Boissonnet <tom.boissonnet@hhu.de> <tom.boissonnet@hotmail.fr>
```

**Review by sbesson — CHANGES_REQUESTED** [review]
One suggestion. Reads fine otherwise

**Review by sbesson — APPROVED** [review]

**Comment by joshmoore** [comment]
* For Steve, I currently have a kaust.edu.sa address
* For Felix, do you mean Meyenhofer?
* The pirate is apparently `Staffan_Holmqvist`

**Comment by jburel** [comment]
* do you have steve.ogg at ...?
* Felix yes
* For Staffan_Holmqvist do you have contact info? I don't

**Pull request merged**